### PR TITLE
OSDOCS-5996: Networking dashboards stub

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-7081

Link to docs preview:
https://69391--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/networking-dashboards

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @jotak @Amoghrd gh

### CODE REVIEW

**Comment by jotak** [comment]
Thanks @JoeAldinger , it's looking good!
Can we also add the mention of the ingress dashboard? cf https://issues.redhat.com/browse/NETOBSERV-1052 / https://issues.redhat.com/browse/OSDOCS-7076
Or is it going to be in another pull request? (I guess it should go in the same doc page)

**Comment by JoeAldinger** [comment]
> Thanks @JoeAldinger , it's looking good! Can we also add the mention of the ingress dashboard? cf [issues.redhat.com/browse/NETOBSERV-1052](https://issues.redhat.com/browse/NETOBSERV-1052) / [issues.redhat.com/browse/OSDOCS-7076](https://issues.redhat.com/browse/OSDOCS-7076) Or is it going to be in another pull request? (I guess it should go in the same doc page)

@jotak my plan was to merge this so that my colleague @jldohmann can then work on the ingress dashboard. So, yes another PR or two is planned. I also _think_ @skrthomas has this [PR](https://github.com/openshift/openshift-docs/pull/69106/files) to attach to this too.

File: networking/networking-dashboards.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[id="network-observability-operator-operator-dashboards"]
+== Network Observability Operator
+If you have the Network Observability Operator installed, you can view network traffic metrics dashboards by selecting the *Netobserv* dashboard from the *Dashboards* dropdown. For more information about metrics available in this *Dashboard* view,
+// 2/9 Network Observability GAs so we will put this back in then.
+//see xref:../network_observability/metrics-alerts-dashboards#network-observability-viewing-metrics-dashboards_metrics-dashboards-alerts[Network Observability metrics dashboards]
+
+[id="general-networking-ovnk-dashboards"]
+== Networking and OVN-Kubernetes dashboard
+You can view both general networking metrics as well as OVN-Kubernetes metrics from the dashboard. To view general networking metrics select *Networking/Linux Subsystem Stats* from the *Dashboards* dropdown. You can view the following general metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
```
Comment (kcarmichael08) [inline networking/networking-dashboards.adoc:19]:
```suggestion
You can view both general networking metrics as well as OVN-Kubernetes metrics from the dashboard. To view general networking metrics, select *Networking/Linux Subsystem Stats* from the *Dashboards* list. You can view the following general metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
```
Not sure if Dashboards is a [list](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list) or [menu](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list) item but I don't think we use the "dropdown" wording.

File: networking/networking-dashboards.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+To view OVN-Kubernetes metrics select *Networking/Infrastructure* from the *Dashboards* dropdown. You can view the following OVN-Kuberenetes metrics: *Networking Configuration*, *TCP Latency Probes*, *Control Plane Resources*, and *Worker resources*.
```
Comment (kcarmichael08) [inline networking/networking-dashboards.adoc:21]:
```suggestion
To view OVN-Kubernetes metrics, select *Networking/Infrastructure* from the *Dashboards* dropdown. You can view the following OVN-Kuberenetes metrics: *Networking Configuration*, *TCP Latency Probes*, *Control Plane Resources*, and *Worker resources*.
```
same issue

File: networking/networking-dashboards.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="networking-dashboards_{context}"]
+= Networking dashboards
+include::_attributes/common-attributes.adoc[]
+:context: networking-dashboards
+
+toc::[]
+
+Some networking metrics are viewable in dashboards within the {product-title} web console, under *Observe* -> *Dashboards*.
```
Comment (kcarmichael08) [inline networking/networking-dashboards.adoc:9]:
```suggestion
Some networking metrics are viewable in dashboards within the {product-title} web console by selecting *Observe* -> *Dashboards*.
```

File: networking/networking-dashboards.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+[id="network-observability-operator-operator-dashboards"]
+== Network Observability Operator
+If you have the Network Observability Operator installed, you can view network traffic metrics dashboards by selecting the *Netobserv* dashboard from the *Dashboards* dropdown. For more information about metrics available in this *Dashboard* view,
```
Comment (kcarmichael08) [inline networking/networking-dashboards.adoc:13]: see other comment about "dropdown"

**Review by kcarmichael08 — COMMENTED** [review]
Minor suggestions; otherwise lgtm!

File: networking/networking-dashboards.adoc (hunk 3 above)
Comment (Amoghrd) [inline networking/networking-dashboards.adoc:9]: Think can remove Some. Just Networking metrics are ... should be good

File: networking/networking-dashboards.adoc (hunk 1 above)
Comment (Amoghrd) [inline networking/networking-dashboards.adoc:19]: The second line onwards can be in a new line for consistency I feel.

File: networking/networking-dashboards.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+== Network Observability Operator
+If you have the Network Observability Operator installed, you can view network traffic metrics dashboards by selecting the *Netobserv* dashboard from the *Dashboards* drop-down list. For more information about metrics available in this *Dashboard* view,
+// 2/9 Network Observability GAs so we will put this back in then.
+//see xref:../network_observability/metrics-alerts-dashboards#network-observability-viewing-metrics-dashboards_metrics-dashboards-alerts[Network Observability metrics dashboards]
+
+[id="general-networking-ovnk-dashboards"]
+== Networking and OVN-Kubernetes dashboard
+You can view both general networking metrics as well as OVN-Kubernetes metrics from the dashboard.
+
+To view general networking metrics, select *Networking/Linux Subsystem Stats* from the *Dashboards* drop-down list. You can view the following general metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
```
Comment (Amoghrd) [inline networking/networking-dashboards.adoc:21]: `You can view the following general metrics` can be changed to `You can view the following networking metrics`

**Pull request merged**